src/library/features/crates/cratefeature.cpp

```diff
@@ -526,37 +594,37 @@ QModelIndex CrateFeature::rebuildChildModel(CrateId selectedCrateId) {
     }
 
     // Append all the newly created TreeItems in a dynamic way to the childmodel
-    m_childModel.insertTreeItemRows(modelRows, 0);
+    m_pChildModel->insertTreeItemRows(modelRows, 0);
 
     // Update rendering of crates depending on the currently selected track
     slotTrackSelected(m_pSelectedTrack);
 
     if (selectedRow >= 0) {
-        return m_childModel.index(selectedRow, 0);
+        return m_pChildModel->index(selectedRow, 0);
     } else {
         return QModelIndex();
     }
 }
 
 void CrateFeature::updateChildModel(const QSet<CrateId>& updatedCrateIds) {
-    const CrateStorage& crateStorage = m_pTrackCollection->crates();
-    for (const CrateId& crateId: updatedCrateIds) {
-        QModelIndex index = indexFromCrateId(crateId);
-        VERIFY_OR_DEBUG_ASSERT(index.isValid()) {
-            continue;
-        }
-        CrateSummary crateSummary;
-        VERIFY_OR_DEBUG_ASSERT(crateStorage.readCrateSummaryById(crateId, &crateSummary)) {
-            continue;
-        }
-        updateTreeItemForCrateSummary(m_childModel.getItem(index), crateSummary);
-        m_childModel.triggerRepaint(index);
-    }
-    if (m_pSelectedTrack) {
-        // Crates containing the currently selected track might
-        // have been modified.
-        slotTrackSelected(m_pSelectedTrack);
-    }
+    // const CrateStorage& crateStorage = m_pTrackCollection->crates();
```

Comment:
Please explain why this code has been commented out and what needs to be done with it.